src/sage/features/databases.py

```diff
@@ -19,43 +19,10 @@
 
 from . import StaticFile, PythonModule
 from sage.env import (
-    CONWAY_POLYNOMIALS_DATA_DIR,
     CREMONA_MINI_DATA_DIR, CREMONA_LARGE_DATA_DIR,
     POLYTOPE_DATA_DIR)
 
 
-class DatabaseConwayPolynomials(StaticFile):
```

Comment:
Normally I think we would say that this API and should not be removed without deprecation. But I don't object to removing it